website/docs/d/edgegateway.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_edgegateway"
+sidebar_current: "docs-vcd-data-source-edgegateway"
+description: |-
+  Provides an edge gateway data source.
+---
+
+# edge gateway
```

Comment:
In general, we need to capitalize words in the headers like "Edge Gateway".
However, in this case, in all the other pages we print the data source name itself, not the caption.

Reply:
fixed